server/server/server.go

```diff
@@ -44,7 +46,11 @@ func (a *Server) Start() error {
 	a.Router.Use(log.RecoveryWithLogger(a.Log))
 
 	store := store.CreateStore()
-	config := &model.Config{} // TODO read from file
+	config, err := readConfig()
+	if err != nil {
+		a.Log.Error("can't read config", log.Err(err))
```

Comment:
Do we need to log here? This error will be logged after Start returns an error.

Reply:
Yes currently it's true, but if we call Start form somewhere else, we might need this.